tsconfig.json

```
@@ -4,10 +4,10 @@
     "target": "es5",
     "lib": ["es5", "es6", "es7", "es2017", "dom"],
     "jsx": "react",
-    "rootDirs": ["src", "stories"],
+    "rootDirs": ["src", "src/stories"],
     "baseUrl": "src",
     "esModuleInterop": true
   },
-  "include": ["typings.d.ts", "src/**/*", "stories/**/*"],
+  "include": ["typings.d.ts", "src/**/*", "src/stories/**/*", "src/stories"],
```

```suggestion
  "include": ["typings.d.ts", "src/**/*"],
```

should match both `src/stories` and `src/stories/**`. can you try to simplify and see if it still works?